docs/ABI.rst

```diff
@@ -830,6 +830,10 @@ types where the metadata itself has unknown layout.)
   global ::= global 'Tm'                 // merged function
   global ::= entity                      // some identifiable thing
   global ::= type type generic-signature? 'T' REABSTRACT-THUNK-TYPE   // reabstraction thunk helper function
+  global ::= entity generic-signature? type type* 'TK' // key path getter
```

Comment:
Is the new demangling and remangling code tested anywhere?

Reply:
We automatically round-trip every symbol name that gets mangled in SILGen through the demangler and remangler.

Reply:
I didn't know that, handy.